src/CommunityToolkit.Maui.Core/Views/SemanticOrderView/MauiSemanticOrderView.ios.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+using Foundation;
+using Microsoft.Maui.Platform;
+using UIKit;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// The native implementation of the <see href="SemanticOrderView"/> control.
+/// </summary>
+public class MauiSemanticOrderView : ContentView, IUIAccessibilityContainer
+{
+	ISemanticOrderView? virtualView;
+	internal ISemanticOrderView? VirtualView
+	{
+		get => virtualView;
+		set
+		{
+			virtualView = value;
+			UpdateViewOrder();
+		}
+	}
+
+	internal void UpdateViewOrder()
+	{
+		var result = GetAccessibilityElements();
+
+		if (result is not null)
```

Comment:
[From my tests](https://sharplab.io/#v2:C4LgTgrgdgNAJiA1AHwAICYAMBYAUBgRj1UwAJUCAWAbmLIoDoAZASygEdbdiCBOACgBEgHg3AIPuCAlFzwA3AIZhSAewBGAK1IBeUlACmAd1IBhJQFsADgBtdADwDy6/hLwBvPKQ+kAaizDAIcpY+Blo6EJaWMO6ewfp2YHC6itouBDDoMADMMJQAvni50ty48oo2oQDiusAAyma6XoEQugDO/KpqUng8AjYMJtDATl24AGZKYLpyAMYAFvylpNOkbKQ2zrie5Hz80yPd+ASZADxswAB8pFW19Y2WzW0mFtb26srqG26bniyjpO3qBg+PwBIIsEIsFphCIbLZbCgEUgqSZyADWXC20Q84xRcwWClIMhWUHeaiB4LiCSSsM8XzhPz+C1IAFJSOgVlDMDT6fCCIjUAB2QkYzz5XBig7EdlPKy2BxqVxY8iZUm6abAAD83l8/kCsVILgA5tVqKQWiaxZjvh5UCrWC1gKcoBdtQZ4olFEaTWaLaEAKJQCCmJJyFTWBh+izAACeTouTgYABUlPahiNckA===), this will never be null, but it will be an empty collection instead

Reply:
👍